sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-block/thin-provisioning-tools

```diff
@@ -0,0 +1,16 @@
+cros_pre_src_unpack_coreos_unpack() {
+  _COREOS_CARGO_SKIP_INHERIT=1
+  source "${CROS_ADDONS_TREE}"/../eclass/coreos-cargo.eclass
```

Comment:
Maybe make `_COREOS_CARGO_SKIP_INHERIT` local? Or maybe make it a parameter passed to `source` like `source "${CROS_ADDONS_TREE}"/../eclass/coreos-cargo.eclass skip-inherit`?

Reply:
made the variable local, argument to source are weird.